xknx/telegram/telegram.py

```diff
@@ -42,23 +42,23 @@ class Telegram:
 
     def __init__(
         self,
-        group_address: GroupAddress = GroupAddress(None),
+        address: GroupAddress = GroupAddress(None),
```

Comment:
Please update the typings so that both `PhysicalAddress` and `GroupAddress` are allowed here

Reply:
Fixed and squashed immediately.